src/CodeGenerator/src/Generator/PhpGenerator/ClassBuilder.php

```diff
@@ -57,13 +70,6 @@ public function addExtend(string $name): self
         return $this;
     }
 
-    public function setImplement($names): self
```

Comment:
This method is not used anyway, and it calls a non-existent method on the Nette ClassType object.